doc/source/whatsnew/v0.1.0.rst

```diff
@@ -2,12 +2,15 @@
 What's new - v0.1.0
 ===================
 
-Features
-========
-
-- Ability to pull atmospheric measurements from ICOS using the ``openghg.client.retrieve_icos``
-command. See the documentation at [ADD DOCS] and tutorial [ADD TUTORIAL] for further information.
+We're happy to announce the first release of OpenGHG to both PyPI and conda.
 
-Fixes
-=====
+Features
+--------
 
+- Standardise measuremnts taken from many different sources - :ref:`Tutorial 1 - Adding observation data<Tutorial 1 - Adding observation data>`
+- Rank observations to ensure use of the best measurements - :ref:`Tutorial 2 - Ranking observations<Tutorial 2 - Ranking observations>`
+- Compare observations with emissions - :ref:`Tutorial 3 - Comparing observations to emissions<Tutorial 3 - Comparing observations to emissions>`
+- Create workflows using high time resolution CO2 data - :ref:`Tutorial 4 - Working with high time resolution CO2<Tutorial 4 - Working with high time resolution CO2>`
+- Retrieve and explore NOAA ObsPack data - :ref:`Tutorial 5 - Exploring the NOAA ObsPack data<Tutorial 5 - Exploring the NOAA ObsPack data>`
```

Comment:
Just a note that I was swapping these around in PR #328

Reply:
Ah yeah, I'll swap those around.